tests/integration/golang/mlflow/run/create_test.go

```diff
@@ -67,19 +155,25 @@ func (s *CreateRunTestSuite) Test_Ok() {
 			},
 		},
 		StartTime:    1234567890,
-		ExperimentID: fmt.Sprintf("%d", *experiment.ID),
+		ExperimentID: fmt.Sprintf("%d", experimentIDInRequest),
 	}
 
 	resp := response.CreateRunResponse{}
+	client := s.MlflowClient.WithMethod(
+		http.MethodPost,
+	).WithRequest(
+		req,
+	).WithResponse(
+		&resp,
+	)
+	if useNamespaceInRequest {
+		client = client.WithNamespace(
```

Comment:
you don't need to assign to `client`. will be enough only have:
````
client.WithNamespace
```

Reply:
that seemed to be the idea of "fluent" methods like this, that they return the struct you are to use.

Reply:
removed